tools/respec2html.js

```diff
@@ -38,6 +42,49 @@ class Logger {
   }
 }
 
+class StaticServer {
+  /**
+   * @param {number} port
+   * @param {string} source
+   */
+  constructor(port, source) {
```

Comment:
I wonder if we should split construction from source validation... they seem to be different concerns.

Reply:
It encapsulates related logic well enough though..

Reply:
Up do you. I don't mind either way.